django/thunderstore/api/cyberstorm/tests/test_user.py

```diff
@@ -0,0 +1,67 @@
+import json
+
+import pytest
+from django.contrib.auth import get_user_model
+from rest_framework.test import APIClient
+
+from thunderstore.core.types import UserType
+from thunderstore.repository.factories import UserFactory
+
+User = get_user_model()
+
+
+@pytest.mark.django_db
+def test_user_delete__success(
+    api_client: APIClient,
+    user: UserType,
+):
+    api_client.force_authenticate(user)
+    response = api_client.post(
+        f"/api/cyberstorm/user/{user.username}/delete/",
+        json.dumps({"verification": user.username}),
+        content_type="application/json",
+    )
+
+    assert response.status_code == 200
+    response_json = response.json()
+    assert response_json["username"] == user.username
+
+    with pytest.raises(User.DoesNotExist) as e:
```

Comment:
I guess sessions, Oauth credentials etc. are automatically deleted via cascades so we don't have to check they're actually gone here?

Reply:
AFAIK yes. In any case, we are deleting the user in the same way as the Django form does, so I'd expect the same result.